examples/r-image/Dockerfile

```diff
@@ -1,111 +1,66 @@
-# This project is licensed under the terms of the Modified BSD License (also known as New or Revised or 3-Clause BSD), as follows:
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
 
-#    Copyright (c) 2001-2015, IPython Development Team
-#    Copyright (c) 2015-, Jupyter Development Team
+ARG OWNER=jupyter
+ARG BASE_CONTAINER=$OWNER/minimal-notebook
+FROM $BASE_CONTAINER
```

Comment:
why update the base image? Can you update the commit to summarize changes including why you had to change the base image?

Reply:
The base image of Python 3.6 had an issue with PyOpenSSL that made it impossible to build. By updating the base image we can rely on a managed image (minimal-notebook) hence reducing the risk of the base image having conflicts with the required packages.

Reply:
Hi @knaresh please review if possible the new commit.